executor.py

```diff
@@ -18,17 +18,21 @@ class SimpleIndexer(Executor):
     def __init__(
         self,
         match_args: Optional[Dict] = None,
+        key_length: int = 64,
+        override_storage_path: str = None,
```

Comment:
Might be better to add `Optional` typing to `override_storage_path: Optional[str] = None`

Reply:
Cool, new to modern type annotated python though